### PR TITLE
build and install LuaRocks

### PR DESCRIPTION
**Pull request by Tinywan**

### CODE REVIEW

File: Setup-OpenResty.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ tar zxpf luarocks-3.3.1.tar.gz
...
+
+make -j8 && sudo make install
+```
+
+> Cannot find the unzip command, just execute the apt installation
+```
+sudo apt install unzip zip
+```
+
 To configure LuaRocks for OpenResty, run this command:
```
Comment (rushitote) [inline Setup-OpenResty.md:28]: Actually, we already have the build and installation documentation of LuaRocks [here](https://github.com/casbin/lua-casbin/blob/master/Setup-OpenResty.md#:~:text=To%20configure%20LuaRocks,sudo%20make%20install) under the configure LuaRocks section. Is there any reason to change this?

Comment (hsluoyz) [reply Setup-OpenResty.md:28]: @rushitote actually we already have: https://github.com/casbin-lua/lua-resty-casbin , can we remove this `Setup-OpenResty.md` now? (we can merge its diff to: https://github.com/casbin-lua/lua-resty-casbin if there are any)

Comment (rushitote) [reply Setup-OpenResty.md:28]: @hsluoyz Yeah, I think we can do that and provide a link to lua-resty-casbin in the README.

Comment (Tinywan) [reply Setup-OpenResty.md:28]: 17cf0924-88c7-4691-8a29-4e11c2073639

Comment (rushitote) [reply Setup-OpenResty.md:28]: @Tinywan Thanks, I will update the setup doc with the unzip dependency.

Comment (hsluoyz) [reply Setup-OpenResty.md:28]: @rushitote thanksZ!

**Comment by hsluoyz** [comment]
@Tinywan see the fixed instructions here: https://github.com/casbin-lua/lua-resty-casbin/pull/13/files

**Comment by Tinywan** [comment]
@Tinywan ok

**Pull request closed without merging**